board/app/controllers/favorite_controller.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+class FavoriteController extends AppController 
+{
+
+ public function set_favorite()
+    {
+        $comment = Favorite::getDatabyCommentId(Param::get('comment_id'));
+        $comment->user_id = $_SESSION['user_id']; 
+        $method = Param::get('method');
+            
+        switch ($method) {
+            case 'add':
+                $comment->addFavorite();
+                break;
+            case 'remove':
+                $comment->removeFavorite();
+                break;
+            default:
+                throw new InvalidArgumentException("{$method} is an invalid parameter");
+                break;
+        }
+        redirect(url('thread/view', array('thread_id' => $comment->thread_id)));
+    }
+
+    /*
+    * Displays list of user's favorite comments
+    */
+    public function favorites() 
```

Comment:
Done po. 
Changed it to display_user_favorite_comments since it's in the controller po :bow:
Thank you.
--> https://github.com/maragatchalian/FoodFacts/commit/b4ebefb9ce00e564b342a7507c91d90278e72497